src/main/kotlin/no/nav/familie/ef/sak/behandling/fremleggsoppgave/Fremleggsoppgave.kt

```diff
@@ -0,0 +1,10 @@
+package no.nav.familie.ef.sak.behandling.fremleggsoppgave
+
+import org.springframework.data.annotation.Id
+import java.util.UUID
+
+data class Fremleggsoppgave(
+    @Id
+    val behandlingId: UUID,
+    val opprettFremleggsoppgave: Boolean
```

Comment:
Burde vi ha noen context om at dette gjelder fremleggsoppgave for inntekt?

Reply:
Er det tilstrekkelig med 
```
data class OpprettFremleggsoppgave(
    @Id
    val behandlingId: UUID,
    val inntekt: Boolean
```

Reply:
En enum e.l tenker du - for å skille på fremtidige fremleggsoppgaver ?

Reply:
ok, satser på å endre opprettFremleggsoppgave til inntekt 👍